client/pages/clubs/[id].tsx

```diff
@@ -1,22 +1,228 @@
-import React from "react";
-import { useRouter } from "next/router";
-import { useAppContext } from "../../context";
+import { useRouter } from 'next/router'
+import React from 'react'
+import styled from 'styled-components'
+import { useAppContext } from '../../context'
+import { desktopFontSize, device, fontWeight, PageTitle } from '../../styles/'
+import { Club } from '../../context/'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faLongArrowAltLeft } from '@fortawesome/free-solid-svg-icons'
+import { BasicEvent } from '../../context/Base'
+import moment from 'moment'
+
+interface BannerProps {
+  backgroundImageUrl: string
+}
+
+const Banner = styled.div`
+  background-image: url(${(props: BannerProps) => props.backgroundImageUrl || ''});
```

Comment:
```suggestion
const Banner = styled.div<BannerProps>`
  background-image: url(${({ backgroundImageUrl }) => backgroundImageUrl || ''});
  background-position: center;
```

We can take advantage of [styled components custom props](https://styled-components.com/docs/api#using-custom-props)

`background-position: center;` keeps the banner vertically centred inside the div